server/rooms.ts

```diff
@@ -1419,6 +1445,7 @@ export class GameRoom extends BasicChatRoom {
 	active: boolean;
 	format: string;
 	auth: {[userid: string]: string};
+	lastTournament: LastTournamentData | null;
```

Comment:
`lastTournament` is a bad name for this; I'd suggest `lastTournamentData`.

Reply:
Got it, made changes.